iphone/Maps/UI/PlacePage/Components/PlacePageInfoViewController.swift

```diff
@@ -134,6 +138,18 @@ class PlacePageInfoViewController: UIViewController {
         self?.delegate?.didPressWebsite()
       }
     }
+    
+    if let wikipedia = placePageInfoData.wikipedia {
+      wikipediaView = createInfoItem("Wikipedia", icon: UIImage(named: "ic_placepage_wiki"), style: .link) { [weak self] in
```

Comment:
Wouldn't it be better to load these strings from data/strings/strings.txt and share between platforms?